azure/pipelines/dotnet/ci.yml

```diff
@@ -15,8 +15,12 @@ parameters:
 - name: DotNetSdkVersion
   type: string
   default: ''
-  displayName: 'Specifc .NET SDK version to use.'
+  displayName: 'Specific .NET SDK version to use.'
 
+- name: azureSubscription
+  type: string
+  default: ''
+  displayName: 'Optional Azure Subscription to run ''dotnet test''.'
```

Comment:
Explain you might want to use this.

Reply:
Something like this?

`  displayName: 'Optional Azure Subscription to run ''dotnet test'' under Azure CLI context. So you can access Azure resources (like App Configuration, Key Vault, etc) with authenticated credentials.''`

Or the explain in a comment (#)?